third_party/nccl.BUILD

```diff
@@ -43,6 +43,44 @@ cc_library(
         "-Iexternal/nccl_archive/src",
         "-O3",
     ] + cuda_default_copts(),
+    linkopts = select({
+        ":android": [
```

Comment:
okay, sounds good!  Let me know when it's in and I'll rlaunch the testing again.